Author: Marcin Kolny <marcin.kolny@gmail.com>
Description: plugins-bad: use flags and libs from gstreamer-gl pkg config file
Origin: https://git.gnome.org/browse/gstreamermm/commit/?id=3a4698e1c6e2b2c6c703f07687cc485ada734d60
Last-update: 2016-11-15
--- a/examples/Makefile.am
+++ b/examples/Makefile.am
@@ -15,7 +15,7 @@
 
 AUTOMAKE_OPTIONS = subdir-objects
 
-examples_cppflags = $(GSTREAMERMM_CFLAGS)
+examples_cppflags = $(GSTREAMERMM_CFLAGS) $(GSTREAMERMM_PLUGINS_BAD_CFLAGS) $(GSTREAMERMM_GL_CFLAGS)
 gstreamermm_includes = -I$(top_builddir)/gstreamer $(if $(srcdir:.=),-I$(top_srcdir)/gstreamer) 
 local_libgstreamermm = $(top_builddir)/gstreamer/gstreamermm/libgstreamermm-$(GSTREAMERMM_API_VERSION).la
 
--- a/gst-plugins-bad/gstreamermm-plugins-bad/Makefile.am
+++ b/gst-plugins-bad/gstreamermm-plugins-bad/Makefile.am
@@ -23,11 +23,11 @@
 local_includes = -I$(top_builddir)/gstreamer $(if $(srcdir:.=),-I$(top_srcdir)/gstreamer)
 local_cppflags = $(binding_includes) $(local_includes) $(binding_cppflags) -DGSTREAMERMM_PLUGINS_BAD_BUILD=1
 
-AM_CPPFLAGS = $(local_cppflags) $(GSTREAMERMM_PLUGINS_BAD_CFLAGS) $(GSTREAMERMM_CFLAGS)
+AM_CPPFLAGS = $(local_cppflags) $(GSTREAMERMM_PLUGINS_BAD_CFLAGS) $(GSTREAMERMM_CFLAGS) $(GSTREAMERMM_GL_CFLAGS)
 AM_CXXFLAGS = $(GSTREAMERMM_WXXFLAGS)
 
 local_libadd = $(top_builddir)/gstreamer/gstreamermm/libgstreamermm-$(GSTREAMERMM_API_VERSION).la
 
 libgstreamermm_plugins_bad_@GSTREAMERMM_API_VERSION@_la_SOURCES = $(binding_sources)
 libgstreamermm_plugins_bad_@GSTREAMERMM_API_VERSION@_la_LDFLAGS = -no-undefined -version-info $(GSTREAMERMM_SO_VERSION)
-libgstreamermm_plugins_bad_@GSTREAMERMM_API_VERSION@_la_LIBADD  = $(GSTREAMERMM_PLUGINS_BAD_LIBS) $(local_libadd)
+libgstreamermm_plugins_bad_@GSTREAMERMM_API_VERSION@_la_LIBADD  = $(GSTREAMERMM_PLUGINS_BAD_LIBS) $(GSTREAMERMM_GL_LIBS) $(local_libadd)
